modules/generate-diagnostic-bundle-using-roxctl-cli.adoc

```diff
@@ -1,19 +1,64 @@
 // Module included in the following assemblies:
 //
 // * configuration/generate-diagnostic-bundle.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="generate-diagnostic-bundle-using-roxctl-cli_{context}"]
 = Generating a diagnostic bundle by using the roxctl CLI
 
 You can generate a diagnostic bundle by using the `roxctl` CLI.
 
+.Prerequisites
+* To generate a diagnostic bundle, you need `read` permission for the `Administration` resource. This is required for versions of the `DebugLogs` resource older than version 3.73.0.
+
+* You have configured the `ROX_PASSWORD` and the `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Set the `ROX_PASSWORD` and the `ROX_CENTRAL_ADDRESS` environment variables:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_PASSWORD=_<rox_password>_ && export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_ <1>
+----
++
+<1> For `<rox_password>`, specify the stackrox administration password.
+
+.Procedure
+* Run the following command to generate a diagnostic bundle:
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" -p "$ROX_PASSWORD" central debug download-diagnostics
+----
+
+////
 .Prerequisites
 * To generate a diagnostic bundle, you need `read` permission for the `DebugLogs` resource.
 
+* You have configured the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Set the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_API_TOKEN=_<api_token>_
+----
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_
+----
+////
+
+////
 .Procedure
 * Run the following command to generate a diagnostic bundle:
 +
 [source,terminal]
 ----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug download-diagnostics
+----
++
+[source,terminal]
+----
 $ roxctl central debug download-diagnostics
```

Comment:
Shouldn't this be dropped, now that we have defined the command in l.57 ($ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug download-diagnostics)?

Right here, the `-e` flag is missing so this will be incomplete.

Reply:
> Shouldn't this be dropped, now that we have defined the command in l.57 ($ roxctl -e "$ROX_CENTRAL_ADDRESS" central debug download-diagnostics)?
> 
> Right here, the `-e` flag is missing so this will be incomplete.

@dhaus67 This command is obsolete, please see https://github.com/openshift/openshift-docs/pull/56709/files#:~:text=Resolve%20conversation-,.Procedure,%2D%2D%2D%2D,-Member